docs/pages/reference/monitoring/audit.mdx

```diff
@@ -77,7 +79,7 @@ $ ls -l /var/lib/teleport/log/
 Teleport Enterprise Cloud manages the storage of audit logs for you. You can
 access your audit logs via the Teleport Web UI by clicking:
 
-**Activity** > **Audit Log**
+**Audit** > **Audit Log**
```

Comment:
Note: Auto backports only to v17 since this change describes the new side nav in v17.